OTLP Exemplars support (#7199)
micrometer-metrics/micrometer: docs/modules/ROOT/pages/implementations/otlp.adoc

----
management.metrics.distribution.percentiles.test.timer=0.9,0.99
----

== Exemplars

> Exemplars are metadata that you can attach to the value of your time series. They can reference data outside of your metrics. A common use case is storing tracing information (`traceId`, `spanId`) but any arbitrary key-value pairs can be attached. Exemplars are not tags/dimensions (attributes in OTLP terminology); they will not increase cardinality since they belong to the values of the time series. Using them allows you to correlate metrics with traces/spans (or other concepts).

Reviewer: I don't think the current docs are consistent or have anything setup to enforce, but I find for things like markdown or asciidoc, having one sentence per line helps make diffs and review easier.

> Exemplars are metadata that you can attach to the value of your time series. They can reference data outside of your metrics. A common use case is storing tracing information (`traceId`, `spanId`) but any arbitrary key-value pairs can be attached. Exemplars are not tags/dimensions (attributes in OTLP terminology); they will not increase cardinality since they belong to the values of the time series. Using them allows you to correlate metrics with traces/spans (or other concepts).

Reviewer: A polish suggestion. Feel free to edit differently.

```suggestion
Exemplars are metadata that you can attach to the value of your metrics time series. They can reference data outside of your metrics. A common use case is cross-referencing tracing information (`traceId`, `spanId`) but any arbitrary key-value pairs can be attached. Exemplars are not tags/dimensions (attributes in OTLP terminology); they will not increase cardinality since they belong to the values of the time series.
```
